pulsar-websocket/src/main/java/org/apache/pulsar/websocket/data/ProducerMessage.java

```diff
@@ -69,4 +70,13 @@ public class ProducerMessage {
 
     // Base64 encoded serialized schema for payload
     public String valueSchema;
+
+    // Compression type. Do not set it if compression is not performed.
+    public String encryptionParam;
+
+    // The size of the payload before compression. Do not set it if compression is not performed.
```

Comment:
Ah, this one is for `uncompressedMessageSize`

Reply:
Fixed